meta-mender-octavo-osd32mp/templates/local.conf.append

```diff
@@ -0,0 +1,68 @@
+
+#
+# Machine Selection
+MACHINE ?= "osd32mp1-emmc-mender"
+
+#
+# Distribution selection
+DISTRO ?= "openstlinux-weston"
+
+#
+# Package Management configuration
+#PACKAGE_CLASSES ?= "package_ipk"
+
+#
+# SDK target architecture
+SDKMACHINE ?= "x86_64"
+
+#
+# Extra image configuration defaults
+EXTRA_IMAGE_FEATURES ?= "debug-tweaks"
+
+#
+# Clean up working directory after build
+# This saves a **lot** of filesystem space
+INHERIT += "rm_work"
+
+# Enable PR server to avoid version-going-backward issue
+#PRSERV_HOST = "localhost:0"
+
+# =========================================================================
+#   Mender Specific
+# =========================================================================
+
+DISTRO_FEATURES:append = " systemd"
```

Comment:
instead of those four lines, you can also just do an `INIT_MANAGER = "systemd"`

Reply:
Thanks for the suggestion, it is much simpler!

I followed the examples shown in [Building for demo](https://docs.mender.io/operating-system-updates-yocto-project/build-for-demo); maybe it can be adjusted for people to see the simpler version.

I kept my version for now because I couldn't test a new image build before pushing the changes.
Still, if you prefer the short form, just let me know and I can push the changed version after I have a chance to rebuild and test on my board.